unfoldingWord/translationCore
# Fixed Unittests

#### This pull request addresses:

* Fixes some issues with the unit tests mostly due to bad test data.
* Also improved multi-book project detection.

#### How to test this pull request:

Make sure all the test pass consistently. It would be best to restart the build on travis several times and make sure each build passes the tests.

> This PR is currently a work in progress.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2815)
